modules/commands.nix

```diff
@@ -8,16 +8,15 @@ with lib;
 let
   commandAttributes = types.submodule {
     options = {
-      command = mkOption {
-        type = types.str;
-        description = "The command to run.";
-      };
+      command = helpers.mkNullOrOption (with types; either str helpers.nixvimTypes.rawLua) ''
+        The command to run.
+      '';
```

Comment:
The previous implementation was forcing the user to provide a value for this option.
Now, this is not the case anymore.
I would rather keep the previous behavior:
```nix
command = mkOption {
  type = with helpers.nixvimTypes; either str rawLua;
  description = "The command to run.";
};
```

Reply:
Ah of course, good spot